herddb-core/src/main/java/herddb/sql/CalcitePlanner.java

```diff
@@ -884,16 +886,16 @@ private PlannerOp planEnumerableSemiJoin(EnumerableSemiJoin op, RelDataType rowT
             fieldNames[i] = col.name;
             columns[i++] = col;
         }
-        return new SemiJoinOp(fieldNames, columns,
-                leftKeys, left, rightKeys, right);
+        return new JoinOp(fieldNames, columns,
+                leftKeys, left, rightKeys, right, generateNullsOnLeft, generateNullsOnRight, false);
```

Comment:
What is 'false'? Can you please add a comment with the name of the argument?
Like:

false /*name of the argument*/

Reply:
Will do.